addons/survey/views/survey_templates.xml

```diff
@@ -476,20 +482,22 @@
                     <t t-set="answer_class" t-if="not has_correct_answer" t-value="''" />
                     <t t-set="answer_class" t-elif="is_correct" t-value="'bg-success'" />
                     <t t-set="answer_class" t-elif="not is_correct" t-value="'bg-danger'" />
-
-                    <label t-att-class="'o_survey_choice_btn mr-2 py-1 px-3 rounded %s %s' % (answer_class, 'o_survey_selected' if answer_line else '')">
-                        <t t-call="survey.survey_selection_key">
-                            <t t-set="selection_key_class" t-value="'position-relative float-left d-flex'"/>
-                        </t>
-                        <input type="checkbox" t-att-value='label.id' class="o_survey_form_choice_item invisible position-absolute"
-                               t-att-name="question.id"
-                               t-att-checked="'checked' if answer_line else None"
-                               t-att-data-selection-key="letters[item_idx] if useKeySelection else ''"/>
-                        <span class="ml-2" t-field='label.value'/>
-                        <i class="fa fa-check-square float-right mt-1 position-relative"></i>
-                        <i class="fa fa-square-o float-right mt-1 position-relative"></i>
-                        <t t-call="survey.question_suggested_value_image"/>
-                    </label>
+                    <div t-att-class="'%s pl-0 pr-2' % ('col-6' if n_answ in [1, 2, 4] else 'col-4')">
+                         <label t-att-class="'col-12 o_survey_choice_btn mr-2 mb-2 py-3 pl-4 pr-5 rounded %s %s' % (answer_class, 'o_survey_selected' if answer_line else '')"
+                                style="height: calc(100% - 0.5rem); line-break: auto; line-break: auto;">
```

Comment:
- Should use scss
- Please add a small comment in the scss file about the calc height thing
- Looks like you have twice "line-break"

(Same for the other template).